GameData/RealismOverhaul/RO_SuggestedMods/HabTech2/RO_HabTech2_TACLS.cfg

```diff
@@ -0,0 +1,1606 @@
+
+
+@PART[ht2_MPLM]:FOR[RealismOverhaul]:NEEDS[TacLifeSupport]
+{
+	@MODULE[ModuleFuelTanks]
+	{
+		TANK
+		{
+			name = CarbonDioxide
+			amount = 0
+			maxAmount = 4000
+		}
+		TANK
+		{
+			name = Oxygen
+			amount = 4000
+			maxAmount = 4000
+		}
+		TANK
+		{
+			name = Waste
+			amount = 0
+			maxAmount = 10
+		}
+		TANK
+		{
+			name = WasteWater
+			amount = 0
+			maxAmount = 60
+		}
+		TANK
+		{
+			name = Food
+			amount = 100
+			maxAmount = 100
+		}
+		TANK
+		{
+			name = Water
+			amount = 75
+			maxAmount = 75
+		}
+		TANK
+		{
+			name = LithiumHydroxide
+			amount = 200
+			maxAmount = 200
+		}
+	}
+
+	MODULE
+	{
+		name = TacGenericConverter
+		converterName = Oxygen Generator (Electrolysis)
+		StartActionName = Start Oxygen Generator (Electrolysis)
+		StopActionName = Stop Oxygen Generator (Electrolysis)
+		tag = Life Support
+		GeneratesHeat = False
+		UseSpecialistBonus = True
+		SpecialistEfficiencyFactor = 0.2
+		SpecialistBonusBase = 0.05
+		ExperienceEffect = ConverterSkill
+		EfficiencyBonus = 1
+		conversionRate = 7.0
+
+		INPUT_RESOURCE
+		{
+			ResourceName = Water
+			Ratio = 0.0000053129
+		}
+
+		INPUT_RESOURCE
+		{
+			ResourceName = ElectricCharge
+			Ratio = 0.5
+		}
+
+		OUTPUT_RESOURCE
+		{
+			ResourceName = Hydrogen
+			Ratio = 0.006612957
+			DumpExcess = True
+		}
+
+		OUTPUT_RESOURCE
+		{
+			ResourceName = Oxygen
+			Ratio = 0.003116887
+			DumpExcess = True
+		}
+	}
```

Comment:
Same comment for each TAC converter module.  Are the top-level parameters the same?  I imagine the resource ratios certainly are.